### PR TITLE
Features: show wp volunteer count and low registration prompt (story #51)

### PR DESCRIPTION
**Pull request by cewing**
@davisagli: This is a work in progress, I am opening a PR to get a bit of code review, though the tests are still borked at this point.

I am specifically interested in questions of access control for the registration descriptions and people needed icons for each work party.  I'm a bit hazy on the requirements and worried I haven't hit them correctly.

### CODE REVIEW

**Comment by cewing** [comment]
oh, wait.  borked tests are on that other branch.  Never mind that comment.

File: src/vms/static/app/workparty.js [hunk 1]
```diff
@@ -7,10 +7,74 @@ var WorkPartyMessageBoard = require('./messageboard');
...
+
+
 var WorkParty = module.exports.WorkParty = React.createClass({
 
 	render: function() {
 		var data = this.props.data;
+		var user = this.props.user;
+		var reg = this.props.reg;
+		var isCL = reg && utils.hasRole(user, reg, 'CL');
+		var isACL = reg && (isCL || utils.hasRole(user, reg, 'ACL'));
```
Comment (davisagli) [inline src/vms/static/app/workparty.js]: Hey, I'm sorry but I take back what I said about using utils.hasRole for this. For showing the counts we don't care if the user is registered as an ACL or CL *for this particular work party*; we only care if they have a sitewide permission level of Assistant Crew Leader, Crew Leader or Admin (a user with ACL permission should be able to see the counts even for work parties they haven't signed up for yet).

So you just want:

    var isCL = (user.role == 'Crew Leader' || user.role == 'Admin');
    var isACL = (user.role == 'Assistant Crew Leader' || isCL);

and we don't need the registration here.

File: src/vms/static/app/workparty.js [hunk 2]
```diff
@@ -25,14 +89,10 @@ var WorkParty = module.exports.WorkParty = React.createClass({
...
 
-		var acls_needed;
-		if (data.alert_acls_needed && data.show_roster_requests) {
-				acls_needed = <div className="ui panel needed"><img src="/static/images/helmet-orange-needed.png" title="ACLs Needed"/></div>;
-		}
 		return (
 			<div className="item js-item" onClick={this.handleClick}>
 				<div className="ui panel status">{status}</div>
-				{acls_needed}
+				{this.props.data.show_roster_requests && <WorkPartyPeopleNeededPanel data={data} />}
```
Comment (davisagli) [inline src/vms/static/app/workparty.js]: The ACL alert should be shown only if the user has a sitewide permission level of Assistant Crew Leader, Crew Leader, or Admin.  The low registration alert should be shown to everyone.

**Comment by davisagli** [comment]
Aside from my inline comments, looks good.

**Comment by cewing** [comment]
Thanks, @davisagli, that's exactly what I was looking for.

**Comment by cewing** [comment]
@davisagli all set for review here.

**Comment by davisagli** [comment]
Merged manually and deployed

**Pull request closed without merging**